big_tests/tests/sm_helper.erl

```diff
@@ -277,7 +277,13 @@ send_initial_presence(User) ->
 
 process_initial_stanza(User) ->
     send_initial_presence(User),
-    escalus:assert(is_presence, escalus:wait_for_stanza(User)).
+    Message = escalus:wait_for_stanza(User),
+    case escalus_pred:is_iq(Message) of
```

Comment:
What kind of IQ can it be, and is it safe to just ignore it?

Reply:
The iq that is being caught here is ping iq. It sometimes arrives before the presence, resulting in tests failure.